ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/services/ApiSubmissionServiceSpec.scala

```diff
@@ -469,6 +468,7 @@ object ApiSubmissionServiceSpec {
       configuration = ApiSubmissionService
         .Configuration(implicitPartyAllocation, enableDeduplication = true),
       metrics = new Metrics(new MetricRegistry),
+      errorCodesVersionSwitcher = new ErrorCodesVersionSwitcher(enableErrorCodesV2 = false),
```

Comment:
Results of `return proper gRPC status codes for DamlLf errors` test with error codes:

V1 (passes):
```
ALA123:
ACTUAL         : |ABORTED: Command interpretation error in LF-DAMLe: Contract could not be found with id ContractId(#cid). Details: N/A.|
EXPECTED PREFIX: |ABORTED|
ACTUAL         : |ABORTED: Command interpretation error in LF-DAMLe: Interpretation error: Error: Update failed due to a duplicate contract key <unit>. Details: N/A.|
EXPECTED PREFIX: |ABORTED|
ACTUAL         : |ABORTED: Command interpretation error in LF-DAMLe: recreated and original transaction mismatch null expected, but null is recreated. Details: N/A.|
EXPECTED PREFIX: |ABORTED|
ACTUAL         : |INVALID_ARGUMENT: Invalid argument: Command interpretation error in LF-DAMLe: unknown package default Package ID.. Details: N/A.|
EXPECTED PREFIX: |INVALID_ARGUMENT|
ACTUAL         : |INVALID_ARGUMENT: Invalid argument: Command interpretation error in LF-DAMLe: Interpretation error: Error: node NodeId(1) (default Package ID:M:T) has no signatories. Details: N/A.|
EXPECTED PREFIX: |INVALID_ARGUMENT|
ACTUAL         : |ABORTED: Could not find a suitable ledger time after 0 retries|
EXPECTED PREFIX: |ABORTED|
```


V2 (fails):
```
ACTUAL         : |ABORTED: CONTRACT_NOT_FOUND(14,0): Contract could not be found with id ContractId(#cid)|
EXPECTED PREFIX: |ABORTED|
ACTUAL         : |ALREADY_EXISTS: DUPLICATE_CONTRACT_KEY_DURING_INTERPRETATION(10,0): Interpretation error: Error: Update failed due to a duplicate contract key <unit>|
EXPECTED PREFIX: |ABORTED|
ACTUAL         : |INTERNAL: An error occurred. Please contact the operator and inquire about the request: <no-correlation-id>|
EXPECTED PREFIX: |ABORTED|
ACTUAL         : |INVALID_ARGUMENT: COMMAND_PREPROCESSING_FAILED(8,0): unknown package default Package ID.|
EXPECTED PREFIX: |INVALID_ARGUMENT|
ACTUAL         : |INVALID_ARGUMENT: DAML_AUTHORIZATION_ERROR(8,0): Interpretation error: Error: node NodeId(1) (default Package ID:M:T) has no signatories|
EXPECTED PREFIX: |INVALID_ARGUMENT|
ACTUAL         : |ABORTED: FAILED_TO_DETERMINE_LEDGER_TIME(2,0): The participant failed to determine the max ledger time for this command: Could not find a suitable ledger time after 0 retries|
EXPECTED PREFIX: |ABORTED|
```


Code excerpt that produced the above:
```
      .map { results: Seq[(Status, Try[Unit])] =>
        results.foreach { case (code: Status, result: Try[Unit]) =>
          inside(result) { case Failure(exception) =>
            println(s"""ACTUAL         : |${exception.getMessage}|
                       |EXPECTED PREFIX: |${code.getCode.toString}|""".stripMargin)
//            exception.getMessage should startWith(code.getCode.toString)
          }
        }
```